src/deadline/client/cli/_groups/queue_group.py

```diff
@@ -48,6 +50,116 @@ def queue_list(**args):
     click.echo(_cli_object_repr(structured_queue_list))
 
 
+@cli_queue.command(name="export-credentials")
+@click.option("--queue-id", help="The queue ID to use.")
+@click.option("--farm-id", help="The farm ID to use.")
+@click.option(
+    "--mode",
+    type=click.Choice(["USER", "READ"], case_sensitive=False),
+    default="USER",
+    help="The type of queue role to assume (default: USER)",
+)
+@click.option(
+    "--output-format",
+    type=click.Choice(["credentials_process"], case_sensitive=False),
+    default="credentials_process",
+    help="Format of the output (default: credentials_process)",
+)
+@click.option("--profile", help="The AWS profile to use.")
+@_handle_error
+def queue_export_credentials(mode, output_format, **args):
+    """
+    Export queue credentials in a format compatible with AWS SDK credentials_process.
+    """
+    start_time = time.time()
+    is_success = True
+    error_type = None
+    # Without this, CodeQL thinks these are unused.
+    _ = is_success, error_type
+
+    # Get a temporary config object with the standard options handled
+    config = _apply_cli_options_to_config(required_options={"farm_id"}, **args)
```

Comment:
Can you add "queue_id" to this required_options, and then reduce some of the error checking below?